docs/memcached.md

```diff
@@ -0,0 +1,14 @@
+# Memchached entries
+
+## `url_prefix:https://github.com/:owner/`
```

Comment:
They actually all have `lp:` prefixed (see `src/server/helpers/memcached.js`).  May be worth prefixing that here and below too for the sake of operators.